examples/shapes/src/main.rs

```diff
@@ -0,0 +1,183 @@
+use anyhow::Result;
```

Comment:
Please add our copyright header to the new file.

```suggestion
// Copyright 2024 the Vello Authors
// SPDX-License-Identifier: Apache-2.0 OR MIT

use anyhow::Result;
```

Reply:
Added!